apps/condo/domains/ticket/utils/serverSchema/TicketChange.js

```diff
@@ -0,0 +1,163 @@
+const _ = require('lodash')
+const { TicketChange } = require('./index')
+const { getById } = require('@core/keystone/schema')
+
+/*
+    TODO(antonal): when this function will be used, strange errors occuring, like "Cannot query field "statusReopenedCounterFrom" on type "TicketChange".',"
+ */
+// const fieldsToTrackInTicketForChanges = () => (
+//     trackableFieldsFrom(Ticket.schema, { except: OMIT_TICKET_CHANGE_TRACKABLE_FIELDS })
+// )
+
+const createTicketChange = async (fieldsChanges, { existingItem, updatedItem, context }) => {
+    const payload = {
+        dv: 1,
+        sender: updatedItem.sender,
+        ticket: { connect: { id: existingItem.id } },
+        ...fieldsChanges,
+    }
+    await TicketChange.create(
+        context.createContext({ skipAccessControl: true }),
+        payload,
+    )
+}
+
+/**
+ * String representation of related item in a single relationship.
+ * Will be displayed in UI in changes history block.
+ * 👉 When a new single relation field will be added to Ticket, new resolver should be implemented here
+ */
+const displayNameResolvers = {
+    'property': async (itemId) => {
+        const item = await getById('Property', itemId)
```

Comment:
it's better don't use getById hack. Can you use context?

Reply:
Why it is better?
Did you mean, `context.executeGraphQL`?

Reply:
Will be implemented in https://doma.atlassian.net/browse/SBERDOMA-391